Dockerfile

```diff
@@ -28,4 +42,5 @@ LABEL org.label-schema.schema-version="1.0.0-rc1" \
     org.label-schema.version=${VERSION}
 ENTRYPOINT [ "/lazydocker" ]
 VOLUME [ "/.config/jesseduffield/lazydocker" ]
-COPY --from=builder /go/src/github.com/jesseduffield/lazydocker/lazydocker /lazydocker
\ No newline at end of file
+ENV PATH=/
+COPY --from=docker-builder /go/src/github.com/docker/cli/build/docker /docker
```

Comment:
Perhaps it would be better to place `docker` binary at `/usr/bin/docker`, which would not require overwriting `PATH` environment variable (hence one less layer).

Reply:
I think `PATH` is not defined on Scratch images. I defined it (in the next commits) as `/bin` and placed the binaries there.